webapp/src/modules/dayConfig.js

```diff
@@ -0,0 +1,42 @@
+// This module allows extension of plugins with dayjs globally,
```

Comment:
I haven't used this pattern but I kinda like it. I may modify it if it messes with static analysis though.